src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -340,6 +340,10 @@
 {%- endblock form_errors -%}
 
 {%- block form_rest -%}
+  {% if render_hook is not defined or render_hook %}
+    {{ renderhook('display' ~ name ~ 'FormRest', { form: form }) }}
```

Comment:
there might be a problem with forms that don't have custom names as the hook name will be not unique. 
For example, many options forms have "form" as form name.

Reply:
Indeed this is something we need to change

Reply:
We should camelize the form name to avoid bugs like https://github.com/PrestaShop/PrestaShop/issues/13594 and https://github.com/PrestaShop/PrestaShop/pull/13580

Reply:
By the no need to call `parent()` or something in this block?

Reply:
Forget this last comment, I didn't see that children were rendered after